diff --git a/modrinth/project.py b/modrinth/project.py
index 275041d..f3a67d1 100644
--- a/modrinth/project.py
+++ b/modrinth/project.py
@@ -22,7 +22,7 @@ class Project:
 
         self.publishedAt = datetime.fromisoformat(data['published'])
         self.updatedAt = datetime.fromisoformat(data['updated'])
-        self.approvedAt = datetime.fromisoformat(data['approved'])
+        # self.approvedAt = datetime.fromisoformat(data['approved'])
         if data['queued'] != None:
             self.queuedAt = datetime.fromisoformat(data['queued'])
 
